2022-01-14 09:50:58,575 INFO    MainThread:126622 [wandb_setup.py:_flush():71] setting env: {}
2022-01-14 09:50:58,576 INFO    MainThread:126622 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 09:50:58,576 INFO    MainThread:126622 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 09:50:58,576 INFO    MainThread:126622 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_095058-22idb0ei/logs/debug.log
2022-01-14 09:50:58,576 INFO    MainThread:126622 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_095058-22idb0ei/logs/debug-internal.log
2022-01-14 09:50:58,577 INFO    MainThread:126622 [wandb_init.py:init():404] calling init triggers
2022-01-14 09:50:58,577 INFO    MainThread:126622 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-14 09:50:58,577 INFO    MainThread:126622 [wandb_init.py:init():460] starting backend
2022-01-14 09:50:58,577 INFO    MainThread:126622 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-14 09:50:58,585 INFO    MainThread:126622 [backend.py:ensure_launched():216] starting backend process...
2022-01-14 09:50:58,588 INFO    MainThread:126622 [backend.py:ensure_launched():221] started backend process with pid: 126681
2022-01-14 09:50:58,589 INFO    MainThread:126622 [wandb_init.py:init():469] backend started and connected
2022-01-14 09:50:58,592 INFO    MainThread:126622 [wandb_init.py:init():533] updated telemetry
2022-01-14 09:50:58,600 INFO    MainThread:126622 [wandb_init.py:init():563] communicating current version
2022-01-14 09:50:58,975 INFO    MainThread:126622 [wandb_init.py:init():568] got version response 
2022-01-14 09:50:58,976 INFO    MainThread:126622 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-14 09:50:59,173 INFO    MainThread:126622 [wandb_init.py:init():606] starting run threads in backend
2022-01-14 09:50:59,231 INFO    MainThread:126622 [wandb_run.py:_console_start():1810] atexit reg
2022-01-14 09:50:59,232 INFO    MainThread:126622 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-14 09:50:59,232 INFO    MainThread:126622 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-14 09:50:59,233 INFO    MainThread:126622 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-14 09:50:59,233 INFO    MainThread:126622 [wandb_init.py:init():633] run started, returning control to user process
2022-01-14 09:50:59,349 INFO    MainThread:126622 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-14 09:50:59,350 INFO    MainThread:126622 [wandb_run.py:_restore():1752] restore
2022-01-14 09:51:01,787 INFO    MainThread:126622 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 09:51:02,023 INFO    MainThread:126622 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 09:51:02,175 INFO    MainThread:126622 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 4862
}

2022-01-14 09:51:02,277 INFO    MainThread:126622 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 4862
}

2022-01-14 09:51:02,378 INFO    MainThread:126622 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4862
  total_bytes: 4862
}

2022-01-14 09:51:02,479 INFO    MainThread:126622 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4862
  total_bytes: 4862
}

2022-01-14 09:51:02,582 INFO    MainThread:126622 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4862
  total_bytes: 4862
}

2022-01-14 09:51:02,684 INFO    MainThread:126622 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4862
  total_bytes: 4862
}

2022-01-14 09:51:02,787 INFO    MainThread:126622 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4862
  total_bytes: 4862
}

2022-01-14 09:51:03,291 INFO    MainThread:126622 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4862
  total_bytes: 4862
}

2022-01-14 09:51:03,679 INFO    MainThread:126622 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4862
  total_bytes: 4862
}
local_info {
}

2022-01-14 09:51:04,732 INFO    MainThread:126622 [wandb_run.py:_append_files():2180] logging synced files
